stable/artifactory-ha/values.yaml

```diff
@@ -487,9 +490,17 @@ artifactory:
     {{- end }}
       database:
         maxOpenConnections: {{ .Values.artifactory.database.maxOpenConnections }}
+      tomcat:
+        connector:
+          maxThreads: 200
+          extraConfig: "acceptCount=\"100\""
```

Comment:
Extract the `extraConfig` also to be a configuration value under `artifactory.tomcat.connector`.

Reply:
Hey eldada, thank you for helping me out. 
I am a bit confused about this one as here (I may be misunderstanding what you mean): https://www.jfrog.com/confluence/display/JFROG/Artifactory+System+YAML
the connector does not have acceptCount as an option but maintenance Connector does.

Also I was told to use extra config here: https://github.com/jfrog/charts/issues/898

Thanks again for your help

Reply:
Yes. This is correct. I might have confused you with my comment. 
You still need the `extraConfig` here. Just its value should come from the values.yaml's `artifactory.tomcat.connector`. 
So here, you'll have 
```
extraConfig: {{ .Values.artifactory.tomcat.connector.extraConfig }}
```

and higher in the values.yaml, you'll have

```
artifactory:
  tomcat:
    connector:
      maxThreads: 200
      extraConfig: `acceptCount="100"`
```
Need to test it... the formatting might need a fixup.

Reply:
This should be resolved now thank you. 
I am a little unsure about my changes to the ReadMe (is my wording correct for acceptCount?)